src/events/2019-09-gtn/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Galaxy Training Network day - (online) CoFest and community call"
+date: '2019-09-12'
+days: 1
+tease: ""
+continent: GL
+location: "Online"
+image: 
+location_url:
+external_url: 
+contact: "Bérénice Batut"
+gtn: false
+---
+
+<img class="float-right" style="max-width: 500px;" src="/events/2019-09-gtn/cofests.png" alt="GTN cofest logo"/>
+
+The Galaxy Training Network is organizing every 3 months one day dedicated to the training community: collaborative work on the training content but also calls with the community.
+
+**The next day will be on the 12th of September.** Anyone who would like to contribute, learn how to contribute to the Galaxy Training Material or just catch up with the community is very welcome to join.
+
+During the day, we will have **2 GTN community calls** to share news about Galaxy and Training, but also just talk together.
+To accommodate different time zones and schedules, there will be 2 calls (with similar discussions) at 8am (UTC + 2) and 4pm (UTC + 2) via [Hangout](https://hangouts.google.com/hangouts/_/calendar/Z2FsYXh5dW5pZnJlaWJ1cmdAZ21haWwuY29t.6nungcj20l9qh2uicb8h5ivpds).
```

Comment:
Did you want to do the timezone link thingy that lets people figure out the time for them?

Reply:
I added them